lib/active_merchant/billing/gateways/transact_pro.rb

```diff
@@ -0,0 +1,231 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    # For more information visit {Transact Pro Services}[https://www.transactpro.lv/business/]
+    #
+    # This gateway was formerly associated with www.1stpayments.net
+    #
+    # Written by Piers Chambers (Varyonic.com)
+    class TransactProGateway < Gateway
+      self.test_url = 'https://gw2sandbox.tpro.lv:8443/gw2test/gwprocessor2.php'
+      self.live_url = 'https://www2.1stpayments.net/gwprocessor2.php'
+
+      class SslConnection < ActiveMerchant::Connection
```

Comment:
Uh, am I missing something here? This class seems useless, and you could just use an `ActiveMerchant::Connection` in your `new_connection` method below